wagtail/admin/templates/wagtailadmin/pages/confirm_delete.html

```diff
@@ -9,11 +9,29 @@
     <div class="nice-padding">
         <p>
             {% trans 'Are you sure you want to delete this page?' %}
-            {% if descendant_count %}
+
+            {% if descendant_count and translation_count %}
+                {% blocktrans count counter=descendant_count %}
+                    Deleting this page will also delete {{ descendant_count }} subpage,
```

Comment:
sorry @KalobTaulien, just updated the messages for delete in https://github.com/wagtail/wagtail/issues/7592#issuecomment-1034790403 -- we're trying to standardise on "child pages" rather than subpages. Not sure how I missed updating this ealier (Unpublish/Move) have the correct messages 🤦‍♂️

Reply:
No problem! I updated these, let me know if I missed anything in the new copy though.